src/net/mod.rs

```diff
@@ -1,14 +1,42 @@
 //! Networking primitives for TCP/UDP communication.
 //!
-//! For OS-specific networking primitives like Unix domain sockets, refer to the [`async_std::os`]
-//! module.
+//! This module provides networking functionality for the Transmission Control and User
+//! Datagram Protocols, as well as types for IP and socket addresses.
```

Comment:
```suggestion
//! Datagram Protocols (TCP and UDP), as well as types for IP and socket addresses.
```

Reply:
Afaik TCP is not commonly described as a "datagram protocol" (it does use packets, but not _datagrams_). I think it's okay to lean on the wording the stdlib provides.